quickstarts/minikube.md

```diff
@@ -1,37 +1,72 @@
-# Starting Minikube
+Minikube provides a local Kubernetes, designed to make it easy to learn and develop for Kubernetes.
+The Kubernetes cluster is started inside a virtual machine, running a Linux kernel and a Docker daemon.
 
-This assumes that you have the latest version of the `minikube` binary, which you can get [here](https://kubernetes.io/docs/setup/minikube/#installation).
+# Installing the dependencies
+
+This quickstart assumes that you have the latest version of the `minikube` binary, which you can get from the [minikube website](https://minikube.sigs.k8s.io/docs/start/).
+
+Minikube requires a container or virtual machine manager.
+The Minikube documentation includes a list of suggested options in the [getting started guide](https://minikube.sigs.k8s.io/docs/start/).
+
+You'll also need the `kubectl` binary, which you can get by following the [`kubectl` installation instructions](https://kubernetes.io/docs/tasks/tools/) from the Kubernetes website.
+
+Once you have all the binaries installed, make sure everything works:
+
+```shell
+# Validate minikube
+minikube version
+
+# Validate kubectl
+kubectl version
+```
```

Comment:
Both commands list the current version that's installed, so in some ways the output is fairly self explanatory. I don't think it's obvious how to best explain what the output should be since the version I have for each won't match the version the reader is using. Happy to hear any suggestions though